client/src/pages/HomePage.tsx

```diff
@@ -0,0 +1,34 @@
+import { NativeStackScreenProps } from "@react-navigation/native-stack";
+import { Button, Text, Image, View } from "../design-system";
+import { RootStackParamList } from "../types/RootStackParamList ";
+import { useScreenSize } from "../hooks/useScreenSize";
+
+export const HomePage = ({
+  navigation,
+}: NativeStackScreenProps<RootStackParamList, "Home">) => {
+  const { height } = useScreenSize();
+
+  return (
+    <View alignItems="center">
+      <Image
+        marginTop={height * 0.25}
```

Comment:
check if you can maybe avoid using this marginTop. you could use flex style in the root component and group the necessary components in order to not use marginTop. i did it locally and it works pretty nice

Reply:
imo using margins is easier, more consistent and stuff doesnt move around too much when adding new things